lib/setup-nexus.sh

```diff
@@ -59,7 +59,7 @@ nexus-setup repositories "${ROOTDIR}/nexus-repositories.yaml"
 skopeo-sync "${ROOTDIR}/docker"
 # Tag SAT image as csm-latest
 sat_image="artifactory.algol60.net/sat-docker/stable/cray-sat"
-sat_version="3.18.0"
+sat_version="3.19.0"
```

Comment:
@denniswalker, we are aware that is not optimal that the version of the container image must be updated in two places. However, we didn't see a quick, low-impact way of specifying it in only one location in this repo. I do not believe it would work to read it from the `docker/index.yaml` file because, AFAIK, that file is not included in the release distribution, so it would not be accessible to this script which runs during CSM installation.

Can we accept this change as-is, and create a follow-up Jira to improve upon this?

Reply:
sure

Reply:
Thanks. I created [CRAYSAT-1541](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1541) for this.